lib/greeting.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// module.exports = exports = {};
+// try naming 'exports' something more random
+// unlike original, it doesn't appear to work without var declaration
+var fishExports = module.exports = {};
+
+fishExports.welcome = function(name) {
+  if (/[^a-z]/i.test(name)) throw new Error('non-alphabetical characters not allowed, or no name provided');
+  if (arguments.length === 0) throw new Error('no name provided');
```

Comment:
Try getting more comfortable with using (!name) approach instead as that is the more common practice.